Arjona/template/simple_functions.py

```diff
@@ -10,3 +10,26 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+		
+# Python program to check if  
+# given number is prime or not 
+  
+num = 11
+  
+# If given number is greater than 1 
+if num > 1: 
+      
+   # Iterate from 2 to n / 2  
+   for i in range(2, num//2): 
+         
+       # If num is divisible by any number between  
+       # 2 and n / 2, it is not prime  
+       if (num % i) == 0: 
+           print(num, "is not a prime number") 
+           break
+   else: 
+       print(num, "is a prime number") 
+  
+else: 
+   print(num, "is not a prime number") 
```

Comment:
If this was a function, these print() statements would need to be changed into returning True or False values.